Update why-bitcoin-is-unique.md (#238)
BitcoinDesign/Guide: guide/getting-started/why-bitcoin-is-unique.md

# Why Bitcoin is unique

> The Internet lets people anywhere instantly exchange ideas and information with people everywhere. Bitcoin takes this further by giving everyone a baked in method for storing and transferring value online.

Reviewer: Is this clear for non-English speakers?

Author: I don't think so, what's the alternative we can use?

Reviewer 2: "...Bitcoin takes this further by giving everyone an internet native method for storing and transferring value..."

Re-referring back to the internet which was introduced in the prior sense more sense to me.

Author: @sqcken what do you think?

Reviewer: ```suggestion
The Internet lets people anywhere instantly exchange ideas and information with people everywhere. Bitcoin takes this further by giving everyone a built-in method for storing and transferring value online.
```

   layout = "float-right-desktop"
%}

Historically, in order to be considered money, it must be usable as each of the following:

1. A medium of exchange
2. A unit of account
3. A store of value

In addition, six fundamental properties define whether or not a currency can actually function as money. Those [properties](https://en.wikipedia.org/wiki/Money#Properties) are:

1. Durability
2. Portability
3. Fungibility
4. Scarcity
5. Divisibility
6. Recognizability

Although bitcoin is purely digital, it meets every classical definition of what makes something money. Instead of relying on physical properties (like gold and silver) or central authorities (like government-issued fiat currencies), bitcoin relies on the world’s most powerful computer network to mathematically enforce the rules that make it the first truly digital form of cash.
</div>

## You own it

When you take cash out of your bank account, it becomes your responsibility. If you lose it, it’s lost. If you transact with it, no middleman can block the transaction. Either way, you are its custodian—this is bitcoin in a nutshell. It’s digital, self-custodial cash.

> Fiat currencies in digital form are different. Online banks and payment processors heavily depend on third-party websites and applications. Are they vetted? Will they responsibly manage your money? Are they even solvent? Tough to say. You just have to trust your bank’s judgment.

Reviewer: I'm personally not a fan of throwing shade at banks. The banking system still works great for many purposes, and regulation is an important factor in stabilizing forces in a society. Also, 99% of people also just have to trust Bitcoin and Bitcoin application developers because they can't verify that the code is legit. My inclination would be to avoid attacking the character of banks and focus on how banking is not suited well for a variety of use cases (typically related to risk calculation and organizational overhead) and how Bitcoin simply works better for those.

Author: I don't think this section in particular throws shadow at banks? I think it actually compares stuff quite well?

Reviewer: It's based on unfounded suspicion that banks are not trustworthy. It's an ad hominem attack - suggesting poor character rather than actually analyzing the trustworthiness of the two systems. It would be different if there were facts about vetting, solvency and a history of not being trustworthy by banks. You could easily write the same paragraph about Bitcoin ("Relies on open-source software. Is it vetted? Will the devs change the code and break Bitcoin? Tough to say...").

When using Bitcoin, you also have to relay on third-party websites and applications that non-technical people have no chance of vetting. So the same argument about whether banks are trustworthy applies to Bitcoin as well (it's just a different type of trust). I'd just leave the argument about trust out, as the section is about the ownership aspect (which is not about trusting the system, but you holding something compared to someone else holding it for you).

That got a bit long, but hope it makes sense. I just think it's a bit hand-wavy.

Author: @GBKS @sqcken  so any concrete suggestions here?  This one is a blocker to get this merged.

Reviewer: My recommendation is to just remove that middle paragraph. The text works without it.

Author: @GBKS The first paragraph talks about fiat in the physical world and the second one complements it by giving the state of things in the digital world, but sure we can remove it, though I still think it gives nice overview on the state of things.

Reviewer: ```suggestion
```
OK, I'll remove it.

Bitcoin combines the custodial benefits of physical cash with the convenience of online banking and payment processing. But unlike your bank, bitcoin has never been hacked. So as long as you responsibly manage your private keys, not only do you own your bitcoin, no one can take them away.

## Global by default

> If you can access the Internet, you can access bitcoin. As of April 2021, roughly [53.6% of the world’s population](https://en.wikipedia.org/wiki/Global_Internet_usage) was online. That means 53.6% of the world already has access to bitcoin. This makes bitcoin the first currency of any kind entirely unbound by national borders. It also makes it the first currency to completely cut out middlemen like currency exchanges, local payment services, and banks.

Reviewer: ```suggestion
If you can access the Internet, you can access bitcoin. As of 2019, roughly [53.6% of the world’s population](https://en.wikipedia.org/wiki/Global_Internet_usage) was online. That means 53.6% of the world already has access to bitcoin. This makes bitcoin the first currency of any kind entirely unbound by national borders. It also makes it the first currency to completely cut out middlemen like currency exchanges, local payment services, and banks.
```
The numbers in the linked Wikipedia article are from 2019.

   layout = "float-right-desktop"
%}

> Bitcoin uses scrambled, pseudonymous strings of letters and numbers in place of personal information, real-life identities, and even usernames. It treats everyone from individuals to corporations and governments the same. No group has the power to censor transactions, shut others out of the network, or confiscate another’s bitcoin. While certain services might discriminate or request personal information, the bitcoin network not only won’t, but can’t.

Reviewer: ```suggestion
Bitcoin uses scrambled, pseudonymous strings of letters and numbers in place of personal information, real-life identities, and even usernames. It treats everyone from individuals to corporations and governments the same. No group has the power to censor transactions, shut others out of the network, or confiscate another’s bitcoin. While certain services might discriminate or request personal information, the Bitcoin network not only won’t, but can’t.
```
